sonar-kotlin-plugin/src/main/java/org/sonarsource/kotlin/checks/CommentedCodeCheck.kt

```diff
@@ -28,8 +28,9 @@ import org.sonar.check.Rule
 import org.sonarsource.kotlin.api.AbstractCheck
 import org.sonarsource.kotlin.api.getContent
 import org.sonarsource.kotlin.converter.KotlinCodeVerifier
+import org.sonarsource.kotlin.converter.KotlinTextRanges.merge
+import org.sonarsource.kotlin.converter.KotlinTextRanges.textRange
 import org.sonarsource.kotlin.plugin.KotlinFileContext
-import org.sonarsource.slang.impl.TextRanges
 
 /**
  * Replacement for [org.sonarsource.slang.checks.CommentedCodeCheck]
```

Comment:
We can get rid of such comments (in all the checks), as these classes do not exist anymore in this repository and we can modify the implementation in the future.